### ISSUE TITLE
Unable to use GJF anymore because it's incompatible with Java 8 + Gradle 7.x

### ISSUE THREAD

**Issue by jimshowalter** [google/google-java-format#602]
For over a year, GJF has been causing deprecation warnings to show up, with Gradle saying it would break in Gradle 7.x.

We had really hoped those would be addressed.

They were, but unfortunately not until *after* GJF was changed to require Java 9 or later.

For projects on Java 8 (which is probably the bulk of all Java projects, ever), we are now faced with either never upgrading past Gradle 6.x, or stopping using GJF.

Would it be possible for the GJF team to take pity on us and release a patched version of GJF that's compatible with both Java 8 and Gradle 7.x?

**Comment by tbroyer** [comment]
Duplicate of #593
